core/capabilities/ccip/configs/solana/chain_writer.go

```diff
@@ -50,104 +54,68 @@ func getCommitMethodConfig(fromAddress string, offrampProgramAddress string, des
 			{
 				Location:      destTokenAddress,
 				WalletAddress: chainwriter.Lookup{AccountLookup: &chainwriter.AccountLookup{Location: receiverAddress}},
-				TokenProgram: chainwriter.Lookup{AccountsFromLookupTable: &chainwriter.AccountsFromLookupTable{
-					LookupTableName: "PoolLookupTable",
-					IncludeIndexes:  []int{6},
-				}},
+				TokenProgram: chainwriter.Lookup{
```

Comment:
Just realized - I messed this up in my original PR, the ATAs should be in the exec context not the commit one